TcUnit-Runner/Program.cs

```diff
@@ -58,6 +61,7 @@ static void Main(string[] args)
                 .Add("v=|VisualStudioSolutionFilePath=", "The full path to the TwinCAT project (sln-file)", v => VisualStudioSolutionFilePath = v)
                 .Add("t=|TcUnitTaskName=", "[OPTIONAL] The name of the task running TcUnit defined under \"Tasks\"", t => TcUnitTaskName = t)
                 .Add("a=|AmsNetId=", "[OPTIONAL] The AMS NetId of the device of where the project and TcUnit should run", a => AmsNetId = a)
+                .Add("T=|Timeout=", "[OPTIONAL] Timeout the process with an error after X minutes", T => Timeout = T)
```

Comment:
is it OK to combine t with T? maybe have something else than T for timeout?

Reply:
**Note to self:** Update documentation of TcUnit.org website

Reply:
You do not like the capital letter? I mean, we could use any other letter, but t is kinda fitting.

Reply:
no, it's more the fact that the letter "t" is already used (and in the BAT-script you can provide both versions for every argument, so a or A for AmsNetId for example)